jinja2/qfdmo/_addresses_partials/map_and_detail.html

```diff
@@ -137,9 +137,9 @@ <h3 class="qfdmo-text-2xl sm:qfdmo-text-4xl qfdmo-text-center text-black qfdmo-m
                             </div>
                         </div>
                     {% endwith %}
-                    {% for acteur in acteurs %}
+                    {% for acteur in acteurs.with_bonus() %}
                         <script type="application/json" data-map-target="acteur">
-                            {{ acteur.json_acteur_for_display(direction=form.initial.direction, action_list=form.initial.action_list, carte=carte) | safe }}
+                            {{ acteur.json_acteur_for_display(direction=form.initial.direction, action_list=form.initial.action_list, carte=carte, bonus=acteur.bonus) | safe }}
```

Comment:
BON, je suis pas hyper fan de ça : j'aimerais ne pas avoir à repasser manuellement la valeur de `bonus` ici, mais à part revoir toute la méthode `acteur.json_acteur_for_display` je n'ai pas de solution. 

Pour plus tard, je pense qu'on pourrait en faire un template_tag (ou équivalent jinja), mais pour le moment ça fait le job

Reply:
pourquoi tu as besoin de le passer en paramètre aors que c'est un champ de acteur ?

**acteur.json_acteur_for_display**(direction=form.initial.direction, action_list=form.initial.action_list, carte=carte, bonus=**acteur.bonus**)

Reply:
Proposition de simplification

```suggestion
                            {{ acteur.json_acteur_for_display(direction=form.initial.direction, action_list=form.initial.action_list, carte=carte) | safe }}
```